ci-scripts/check_api_server.sh

```diff
@@ -8,7 +8,7 @@ cd $DIR/..
 $DIR/gen_api_server.sh > /dev/null
 
 # Check if anything changed, the README contains a generation date, ignore that file.
-changes=$(git status --porcelain api-server | grep -v README.md)
+changes=$(git status --porcelain api-server | grep -v README.md | grep -v Cargo.toml)
```

Comment:
why was this changed? Does the Cargo.toml file have something dynamic in it?

Reply:
When I was trying this, release would touch those files, but I think I can adjust release to only touch the yaml files, and not mess with this file.

Reply:
is that because the release wants to update the version in the Cargo.toml file? If so how does that change get committed into the repo?

Reply:
Yeah, release wants to update all Cargo.toml files. I adjusted the release script to exclude the generated packages. Release also commits those changes before it does a tag.